webpack.config.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
```

Comment:
should use gulp-babel instead of webpack, webpack is only for final bundles or wacky libraries